src/chains/ethereum/ethereum/package.json

```diff
@@ -27,7 +27,7 @@
   "scripts": {
     "docs.build": "npx shx rm -rf ./lib/docs ./lib/api.json && npm run docs.typedoc",
     "docs.typedoc": "typedoc --options ./typedoc.json --readme ./README.md --out ../../../../docs/typedoc --json ../../../../docs/typedoc/api.json src/api.ts",
-    "docs.preview": "ws --open --port 3010 --directory ../../../../docs",
+    "docs.preview": "ws --open --hostname localhost --port 3010 --directory ../../../../docs",
```

Comment:
Fair enough - I don't use WSL, but trust that this does help :)

Reply:
Haha. 127.0.0 doesn't make it to wsl from windows by default, but localhost does.